dbms/src/Common/setThreadName.cpp

```diff
@@ -77,5 +77,15 @@ std::string getThreadName()
 
 std::string getThreadNameAndID()
 {
-    return getThreadName() + "_" + DB::toString(pthread_self());
+    uint64_t thread_id;
+#if defined(__APPLE__)
```

Comment:
Can we just use `std::this_thread::get_id()`?

Reply:
> Can we just use `std::this_thread::get_id()`?

`std::this_thread::get_id()` may not a uint64.  https://stackoverflow.com/questions/7432100/how-to-get-integer-thread-id-in-c11